.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+// Development container for the Data Safe Haven project
+{
+	"name": "Turing Data Safe Haven",
+	"build": {
+		"context": "..",
```

Comment:
This is cool. I don't think Docker can include files from outside parent-of-Dockerfile natively :)

Reply:
This is cool. I don't think Docker can include files from outside parent-of-Dockerfile natively :). Merge whenever you're ready @craddm .